### DISCUSSION TITLE
How to view history by latest items

### DISCUSSION THREAD

**Discussion by LinArcX** [pystardust/ytfzf#689] [Q&A]
I'm using this command to see my watching history:
`~/bin/scripts/yt.sh -H -t -c`

The problem is that it shows items from old to new. so the new ones are at the bottom. I want to reverse this order. Any idea?

**Comment by LinArcX** [comment]
There is a `--tac` option for fzf or even we can use it as a separate command in Linux:

`echo -e "hi\nbye" | tac | fzf`

But I don't know how can i apply it to ytfzf.

**Reply by Euro20179** [reply]
I think you could simply do `FZF_DEFAULT_OPTS=--tac ~/bin/scripts/yt.sh -H -t -c`

FZF_DEFAULT_OPTS is an environment variable that contains options to always apply to fzf.

**Reply by LinArcX** [reply]
but it will affect other usages of mine from fzf. I have lots of other personal scripts that use fzf and I don't want to reverse their orders.

Maybe i can create a feature request for this one. thanks a lot.

**Reply by Euro20179** [reply]
> but it will affect other usages of mine from fzf. I have lots of other personal scripts that use fzf and I don't want to reverse their orders.

If you use it exactly the way i showed you it will not change other uses of fzf, however if you export FZF_DEFAULT_OPTS in your bashrc (or shell equivelent) it will change fzf's behavior in general.
